res_syntax/tests/ppx/react/expected/defaultValueProp.res.txt

```diff
@@ -0,0 +1,40 @@
+module C0 = {
+  type props<'a, 'b> = {a?: 'a, b?: 'b}
+  @react.component
+  let make = (props: props<'a, 'b>) => {
+    let b = switch props.b {
```

Comment:
Given the error message I'm getting "The module or file props can't be found." this is probably a path `A.foo` where `A` happens to be `props` and not a field access, which it should be.

Reply:
Field access is `Pexp_field`.

Reply:
Oops! Correct.